ci-operator/jobs/openshift/cluster-openshift-apiserver-operator/openshift-cluster-openshift-apiserver-operator-master-presubmits.yaml

```diff
@@ -425,7 +425,6 @@ presubmits:
       ci-operator.openshift.io/prowgen-controlled: "true"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-cluster-openshift-apiserver-operator-master-verify-deps
-    optional: true
```

Comment:
this looks weird.  Doesn't this mean the job would block now?

Reply:
In general, from what I was talking with @tnozicka it's better to just remove the job at all, what's the point of it running if it always fails after switching to go.mod

Reply:
yes, drop the check, optionals don't work and it will always fail -we will bring it back when we teach it gomod